### PR TITLE
Merge develop into master

### PR DESCRIPTION
**Pull request by karojg**

### CODE REVIEW

File: app/Http/Controllers/UrlController.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	/**
+	 * Inits the encode process
+	 */
+  public function encode()
+  {
+    // Call the Model, that will call the Service
+    $urlModel = $this->model->service_call($this->request);
+
+    return $urlModel;
```
Comment (rvalenciano) [inline app/Http/Controllers/UrlController.php]: You should return a json structure with the encoded url.

File: app/Models/Url.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   */
+  protected $fillable = [
+    'url',
+    'url_encoded'
+  ];
+
+  /**
+	 * Executes the call between the model and service
+	 */
+  public function service_call($string)
```
Comment (rvalenciano) [inline app/Models/Url.php]: Rename this to `encode_url`, let's avoid using generic names like `object`, `data`, `service`, `class`, `index`, etc and think about the business domain.

File: app/Http/Controllers/UrlController.php (hunk 1 above)
Comment (karojg) [reply app/Http/Controllers/UrlController.php]: Change done. Method returns a json response.

File: app/Models/Url.php (hunk 2 above)
Comment (karojg) [reply app/Models/Url.php]: Method renamed.

**Pull request merged**